extensions/INTEL/SPV_INTEL_variable_length_array.asciidoc

```diff
@@ -0,0 +1,264 @@
+SPV_INTEL_variable_length_array
+===============================
+
+:capability_token                 : pass:normal[5817]
+:capability_untyped_token         : pass:normal[6243]
+:OpVariableLengthArrayINTEL_token : pass:normal[5818]
+:OpSaveMemoryINTEL_token          : pass:normal[5819]
+:OpRestoreMemoryINTEL_token       : pass:normal[5820]
+:OpUntypedVariableLengthArrayINTEL_token : pass:normal[6244]
+
+:untyped_ptr_khr_url: https://github.com/KhronosGroup/SPIRV-Registry/blob/main/extensions/KHR/SPV_KHR_untyped_pointers.asciidoc
+
+== Name Strings
+
+SPV_INTEL_variable_length_array
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+== Contributors
+
+- Alexey Sotkin, Intel +
+- Ben Ashbaugh, Intel +
+- Vyacheslav Zakharin, Intel +
+- Michael Kinsner, Intel +
+- Dmitry Sidorov, Intel +
+- Viktoria Maksimova, Intel +
+- Victor Lomuller, Codeplay +
+
+== Notice
+
+Copyright (c) 2025 Intel Corporation.  All rights reserved.
+
+== Status
+
+* Complete
+
+== Version
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2025-03-14
+| Revision           | 3
+|========================================
+
+== Dependencies
+
+This extension is written against the SPIR-V Specification,
+Version 1.6 Revision 5.
+
+This extension interacts with {untyped_ptr_khr_url}[*SPV_KHR_untyped_pointers*] extension.
+
+This extension requires SPIR-V 1.0.
+
+== Overview
+
+This extension allows to allocate local arrays whose number of elements is
+unknown at compile time. This is useful for implementing high-level language
+features like C99 variable length arrays.
+
+== Extension Name
+
+To use this extension within a SPIR-V module, the appropriate *OpExtension* must
+be present in the module:
+
+----
+OpExtension "SPV_INTEL_variable_length_array"
+----
+
+== Modifications to the SPIR-V Specification, Version 1.6
+
+=== Capabilities
+
+Modify Section 3.31, Capability, adding rows to the Capability table:
+
+--
+[options="header"]
```

Comment:
Editorial: setting the column widths explicitly causes this to render a bit nicer:
```suggestion
[cols="^.^2,16,15",options="header",width = "100%"]
```